test/install/install.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
why we are removing the `e2e` component of the path?

Reply:
will move it back

Reply:
done